app/Libraries/Markdown/StyleBlock/Parser.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+// See the LICENCE file in the repository root for full licence text.
+
+namespace App\Libraries\Markdown\StyleBlock;
+
+use League\CommonMark\Block\Parser\BlockParserInterface;
+use League\CommonMark\ContextInterface;
+use League\CommonMark\Cursor;
+
+class Parser implements BlockParserInterface
+{
+    public function parse(ContextInterface $context, Cursor $cursor): bool
+    {
+        if ($cursor->match('#^\{{3}[ \t]*#') === null) {
```

Comment:
in removing the space check I used `starts_with` instead of regex to test for `{{{`, so I didn't add a single-character check before it. lmk if you still want taht though. I don't know how to test performance at this level 🤔